### PR TITLE
Replace nemesis to k8s style

### PR DESCRIPTION
**Pull request by mapleFU**
Signed-off-by: mapleFU <1506118561@qq.com>

### What problem does this PR solve? 

Replace nemesis to k8s style.

```
KUBECONFIG=~/.kube/config ./bin/chaos-tidb -namespace fxw -hub hub.pingcap.net -nemesis random_kill
```

### What is changed and how does it work?

Change nemesis code to golang k8s style.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

Side effects

 - Breaking backward compatibility

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/control/control.go [hunk 1]
```diff
@@ -158,22 +158,22 @@ func (c *Controller) syncExec(f func(i int)) {
 func (c *Controller) setUpDB() {
 	log.Printf("begin to set up database")
 	c.syncExec(func(i int) {
```
Comment (you06) [inline pkg/control/control.go]: `syncExec` will trigger input function for every pod, includes tikv, pd, tidb, we should set up db for tidb pod only here. A further problem is that all tidb pods are under a same nodeport service which runs load balance, we should not set up db for the same db address multiple times.

File: pkg/core/nemesis.go [hunk 2]
```diff
@@ -16,9 +23,9 @@ type Nemesis interface {
 	// TearDown(ctx context.Context, node string) error
 
 	// Invoke executes the nemesis
-	Invoke(ctx context.Context, node cluster.Node, args ...string) error
+	Invoke(ctx context.Context, node cluster.Node, chaosNS string, args ...string) error
```
Comment (mahjonp) [inline pkg/core/nemesis.go]: Adding `chaoNS` isn't necessary.

File: pkg/control/control.go [hunk 3]
```diff
@@ -139,7 +138,7 @@ ROUND:
 	nemesisWg.Wait()
 
 	c.tearDownClient()
-	c.tearDownDB()
+	//c.tearDownDB()
```
Comment (mahjonp) [inline pkg/control/control.go]: revert?

File: pkg/control/control.go [hunk 4]
```diff
@@ -78,7 +78,6 @@ func (c *Controller) Close() {
 
 // Run runs the controller.
 func (c *Controller) Run() {
-	// c.setUpDB()
```
Comment (mahjonp) [inline pkg/control/control.go]: uncomment?

File: pkg/nemesis/nemesis.go [hunk 5]
```diff
@@ -2,34 +2,49 @@ package nemesis
...
+		return err
+	}
+	log.Printf("Creating pod-kill with node %s(ns:%s)\n", node.PodName, node.Namespace)
+	return podChaos(ctx, c, node.Namespace, node.Namespace, node.PodName, v1alpha1.PodFailureAction)
 }
 
 func (kill) Recover(ctx context.Context, node cluster.Node, args ...string) error {
-	db := core.GetDB(args[0])
-	return db.Start(ctx, node)
+	c, err := createClient()
```
Comment (mahjonp) [inline pkg/nemesis/nemesis.go]: inject Chaos client into kill struct?

Comment (mapleFU) [reply pkg/nemesis/nemesis.go]: emm, good idea.

File: cmd/tidb/main.go [hunk 6]
```diff
@@ -29,6 +29,7 @@ var (
 	checkerNames = flag.String("checker", "porcupine", "checker name, eg, porcupine, tidb_bank_tso")
 	pprofAddr    = flag.String("pprof", "0.0.0.0:8080", "Pprof address")
 	namespace    = flag.String("namespace", "tidb-cluster", "test namespace")
+	//chaosNamespace = flag.String("chaos-ns", "chaos-testing", "test chaos namespace")
```
Comment (you06) [inline cmd/tidb/main.go]: Why not set a flag for chaos namespace?

File: pkg/core/nemesis.go [hunk 7]
```diff
@@ -105,7 +115,8 @@ func (NoopNemesisGenerator) Generate(nodes []cluster.Node) []*NemesisOperation {
 	ops := make([]*NemesisOperation, len(nodes))
```
Comment (you06) [inline pkg/core/nemesis.go]: Does this means the amount of nemesis is equal to amount of nodes?

Comment (mapleFU) [reply pkg/core/nemesis.go]: @you06 Sure, you can take a look at the code below.

Comment (mahjonp) [reply pkg/core/nemesis.go]: Yes, NoopNemesisGenerator is a faker generator, `NoopNemesis` does nothing actually.

**Pull request merged**